VimeoUpload/Upload/Model/VideoSettings.swift

```diff
@@ -96,4 +112,16 @@ class VideoSettings: NSObject
         aCoder.encodeObject(self.users, forKey: "users")
         aCoder.encodeObject(self.password, forKey: "password")
     }
+    
+    // MARK : String Methods
+    
+    func trim(text: String?) -> String
+    {
+        if let constText = text
+        {
+            return constText.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceCharacterSet())
+        }
+        
+        return ""
```

Comment:
Let's make the return type optional as well (I think it'd be weird if trimming nil returns something other than nil). Then we can condense this into a single line. After that let's merge this in!